### PR TITLE
feat(manage-variables): create manage-variable add, edit markup

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [ ] `Error / Warning / Lint / Type`

### Description
SSIA

Sorry for the long code. It's just markup.

image

### CODE REVIEW

File: src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue [hunk 1]
```diff
@@ -0,0 +1,266 @@
+<template>
+    <div class="dashboard-manage-variable-overay">
+        <p-pane-layout class="page-wrapper">
+            <!--song-lang-->
```
Comment (yuda110) [inline src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue:4]: todo: song-lang is over! 🥲 You have to add babel yourself

File: src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+                { name: 'name', label: 'Name' },
+                { name: 'selectionType', label: 'Selection Type' },
+                { name: 'variableType', label: 'Variable Type' },
+                { name: 'use', label: 'Use' },
+                { name: 'options', label: 'Options' },
+                { name: 'edit', label: ' ' },
+                { name: 'delete', label: ' ' },
+            ],
+            selectionType: computed(() => ({
+                SINGLE_SELECT: i18n.t('Single '),
```
Comment (yuda110) [inline src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue:157]: nitpick: + `MULTI_SELECT` ?

**Review by yuda110 — APPROVED** [review]
LGTM!!!
Please add translations :)

File: src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue (hunk 2 above)
Comment (piggggggggy) [reply src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue]: Thank!! I miss it..!

**Pull request merged**